timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -427,18 +427,19 @@ def tune_model_hyperparameters(
         ):
             default_num_trials = 10
 
+        tuning_start_time = time.time()
         with disable_tqdm():
-            hpo_models, hpo_results = model.hyperparameter_tune(
+            hpo_models, _ = model.hyperparameter_tune(
```

Comment:
`hpo_models` contains all the information that we need to report in the same format regardless of the backend, so we ignore `hpo_results` (which is backend-dependent)